### DISCUSSION TITLE
How can one make sure not to miss a new release (e.g. through feed, push-notifications, mailing list...) ?

### DISCUSSION THREAD

**Discussion by CTBlaze** [monocasual/giada#567] [Q&A] [answered]
n/t

**Comment by gvnnz** [comment, marked as answer]
You can grab the RSS for GitHub releases [here](https://github.com/monocasual/giada/releases.atom). Also, we are about to launch a Discord server for public announcements and end-user communication. Stay tuned for the latter :)

**Reply by CTBlaze** [reply]
Discord?  *yay*
excellent, way better than this board, for actual discussions etc.
(I am running my own special interest discord server, too)
